packages/analyticsdx-template-lint/test/testutils.ts

```diff
@@ -95,7 +95,8 @@ function newVscodeAjv(options?: AjvOptions): Ajv {
     'deprecationMessage',
     'doNotSuggest',
     'enumDescriptions',
-    'patternErrorMessage'
+    'patternErrorMessage',
+    'errorMessage'
```

Comment:
`errorMessage` is a vscode-specific extension to json-schema, so we have to tell ajv about it for the tests.